tests/core/go_test/testmain_bad_test.go

```diff
@@ -0,0 +1,13 @@
+// +build !good
```

Comment:
Remove this constraint? Not sure why it's here.

Also, please add a copyright notice (copy from another file).

Reply:
I copied this from `tags_bad_test.go` - I thought that this was how you mark tests as failing (since that test literally calls `t.Fail()`).

I saw your pointer to a proper fix in the comment below, thanks!